<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgQuotesSaveResult</class>
 <widget class="QWidget" name="wdgQuotesSaveResult">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Report of saved quotes</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>64</width>
           <height>64</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="../images/xulpymoney.qrc">:/xulpymoney/save.png</pixmap>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTabWidget" name="tab">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tabAdded">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/save.png</normaloff>:/xulpymoney/save.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Added</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="mqtwObjects" name="mqtwAdded" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabUpdated">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/editar.png</normaloff>:/xulpymoney/editar.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Updated</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="mqtwObjects" name="mqtwUpdated" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabIgnored">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/study.png</normaloff>:/xulpymoney/study.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Ignored</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="mqtwObjects" name="mqtwIgnored" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tabErrors">
        <attribute name="icon">
         <iconset resource="../images/xulpymoney.qrc">
          <normaloff>:/xulpymoney/button_cancel.png</normaloff>:/xulpymoney/button_cancel.png</iconset>
        </attribute>
        <attribute name="title">
         <string>Errors</string>
        </attribute>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="mqtwObjects" name="mqtwErrors" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mqtwObjects</class>
   <extends>QWidget</extends>
   <header location="global">myqtablewidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../images/xulpymoney.qrc"/>
 </resources>
 <connections/>
</ui>
